{
  "uuid": "8cb307df-cf24-4328-b7bf-b9faabd971f2-tr-0-1003",
  "historyId": "WhiteCase.CompApp.UiTests.Scenarios.Sample_AllureReportTypes+TestClass.SimpleTestFailed",
  "fullName": "WhiteCase.CompApp.UiTests.Scenarios.Sample_AllureReportTypes+TestClass.SimpleTestFailed",
  "labels": [
    {
      "name": "thread",
      "value": "NonParallelWorker"
    },
    {
      "name": "host",
      "value": "AGEORGE01"
    },
    {
      "name": "testClass",
      "value": "WhiteCase.CompApp.UiTests.Scenarios.Sample_AllureReportTypes+TestClass"
    },
    {
      "name": "testMethod",
      "value": "SimpleTestFailed"
    },
    {
      "name": "package",
      "value": "WhiteCase.CompApp.UiTests.Scenarios.Sample_AllureReportTypes+TestClass"
    },
    {
      "name": "owner",
      "value": "unickq"
    },
    {
      "name": "suite",
      "value": "FailedSuite"
    },
    {
      "name": "subSuite",
      "value": "DoesNotThrow"
    },
    {
      "name": "subSuite",
      "value": "AllureReportTypeSamples"
    }
  ],
  "links": [
    {
      "name": "Google",
      "url": "https://google.com",
      "type": "link"
    }
  ],
  "name": "SimpleTestFailed",
  "status": "failed",
  "statusDetails": {
    "known": false,
    "muted": false,
    "flaky": false,
    "message": "  Expected: No Exception to be thrown\r\n  But was:  <System.Exception: I'm an exception\r\n   at WhiteCase.CompApp.UiTests.Scenarios.Sample_AllureReportTypes.TestClass.<>c.<SimpleTestFailed>b__3_0() in C:\\Users\\Ajith.George\\Desktop\\STUDY\\Selenium-WebDriver\\Reporting\\NunitWithAllureReporting_Version1\\NunitWithAllureReporting_Version1\\Sample_AllureReportTypes.cs:line 54\r\n   at NUnit.Framework.Constraints.ThrowsConstraint.VoidInvocationDescriptor.Invoke()\r\n   at NUnit.Framework.Constraints.ThrowsConstraint.ExceptionInterceptor.Intercept(Object invocation)>\r\n",
    "trace": "   at WhiteCase.CompApp.UiTests.Scenarios.Sample_AllureReportTypes.TestClass.SimpleTestFailed() in C:\\Users\\Ajith.George\\Desktop\\STUDY\\Selenium-WebDriver\\Reporting\\NunitWithAllureReporting_Version1\\NunitWithAllureReporting_Version1\\Sample_AllureReportTypes.cs:line 52\r\n"
  },
  "stage": "finished",
  "description": "OLOLO\n",
  "steps": [
    {
      "name": "SimpleTestFailed",
      "status": "failed",
      "statusDetails": {
        "known": false,
        "muted": false,
        "flaky": false,
        "message": "  Expected: No Exception to be thrown\r\n  But was:  <System.Exception: I'm an exception\r\n   at WhiteCase.CompApp.UiTests.Scenarios.Sample_AllureReportTypes.TestClass.<>c.<SimpleTestFailed>b__3_0() in C:\\Users\\Ajith.George\\Desktop\\STUDY\\Selenium-WebDriver\\Reporting\\NunitWithAllureReporting_Version1\\NunitWithAllureReporting_Version1\\Sample_AllureReportTypes.cs:line 54\r\n   at NUnit.Framework.Constraints.ThrowsConstraint.VoidInvocationDescriptor.Invoke()\r\n   at NUnit.Framework.Constraints.ThrowsConstraint.ExceptionInterceptor.Intercept(Object invocation)>\r\n",
        "trace": "   at WhiteCase.CompApp.UiTests.Scenarios.Sample_AllureReportTypes.TestClass.SimpleTestFailed() in C:\\Users\\Ajith.George\\Desktop\\STUDY\\Selenium-WebDriver\\Reporting\\NunitWithAllureReporting_Version1\\NunitWithAllureReporting_Version1\\Sample_AllureReportTypes.cs:line 52\r\n"
      },
      "stage": "finished",
      "steps": [],
      "attachments": [
        {
          "name": "Console Output",
          "source": "600849becf9f45efabcea52f74e22e19-attachment.txt",
          "type": "text/plain"
        }
      ],
      "parameters": [],
      "start": 1568352091324,
      "stop": 1568352091392
    }
  ],
  "attachments": [],
  "parameters": [],
  "start": 1568352091321,
  "stop": 1568352091399
}